packages/component-library/draft/Kaizen/Table/Table.tsx

```diff
@@ -109,10 +129,17 @@ export const TableRow: TableRow = ({ children }) => (
 )
 
 type TableRowCell = React.FunctionComponent<{
```

Comment:
@sebpearce and I were discussing and think that this could be confusing for consumers having width as a percentage and allowing a flex string. WDYT of annotating so it's clear when people use this component?

```suggestion
/**
 * @param width value between 1 and 0, calculated as a percentage
 * @param flex CSS flex shorthand as a string
 */
type TableRowCell = React.FunctionComponent<{
```

Reply:
Like this :D 

<img width="513" alt="image" src="https://user-images.githubusercontent.com/1621182/67821102-ea918f80-fb0f-11e9-9aaf-676a5bdc6bae.png">